homeworkWeekend9/src/ba/bitcamp/homewirkWeekend9/HTTPImageServer/Client.java

```diff
@@ -0,0 +1,63 @@
+package ba.bitcamp.homewirkWeekend9.HTTPImageServer;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.net.Socket;
+import java.net.UnknownHostException;
+
+/**
+ * 
+ * @author Gordan
+ * 
+ *         This class represents one client that connects to HTTP server and
+ *         sends one picture to that server.
+ */
+public class Client {
+
+	// Constants
+	public static final int PORT = 8000;
+	public static final String LOCALHOST = "localhost";
+	private static final String INITIAL_MESSAGE = "PUT";
+
+	public static void main(String[] args) {
```

Comment:
Again, class has no structure. Putting everything into 'main' is not a correct approach.

Main method should be used only for creating instance and calling a 'run' or similar method.